docs/reconciliation/README.md

```diff
@@ -0,0 +1,43 @@
+# Reconciliation
+
+Synthesized compositions are reconciled into real k8s resources by the `eno-reconciler` process.
+
+## Opacity
+
+Eno is designed to treat the resources it manages as completely opaque - it doesn't "understand" their schema, infer dependencies, etc..
+
+There is one exception to this rule: CRDs are always reconciled before CRs of the kind they define.
+
+## Updates
+
+By default, Eno uses [server-side apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) with `--force-conflicts=true` to write resources it manages.
+
+Exceptions:
+
+- The eno-reconciler process can fall back to client-side three-way merge patch by setting `--disable-ssa`
+- Merge can be disabled for a resource by setting the `eno.azure.io/replace: "true"` annotation (a full `update` request will be used instead of a `patch`)
+- All updates can be disabled for a resource by setting the `eno.azure.io/disable-updates: "true"` annotation
+
+## Deletion
+
+Resources are automatically deleted if they are no longer synthesized (returned by the synthesizer) for a given composition.
+
+> This can be disabled by setting the `eno.azure.io/deletion-strategy: orphan` annotation on the composition.
```

Comment:
Is this true? I was under the impression that this only applied to the deletion of the composition itself

Reply:
Good catch! Fixed.